add parameter to Microsoft defender cloud apps event collector (#34764)
demisto/content: Packs/MicrosoftCloudAppSecurity/ReleaseNotes/2_1_63.md

#### Integrations

##### Microsoft Defender for Cloud Apps Event Collector

> - Added support for choosing which event types to fetch by adding a new parameter ***Event types to fetch***.

Reviewer: ```suggestion
- Added support for choosing event types to fetch, using a new ***Event types to fetch*** parameter.
```
